cluster/gce/gci/configure-helper.sh

```diff
@@ -1245,6 +1245,12 @@ current-context: service-account-context
 EOF
 }
 
+function create-node-problem-detector-kubeconfig-from-kubelet {
+  echo "Creating node-problem-detector kubeconfig from /var/lib/kubelet/kubeconfig"
+  mkdir -p /var/lib/node-problem-detector
+  cp /var/lib/kubelet/kubeconfig /var/lib/node-problem-detector/kubeconfig
```

Comment:
Why copy? Why not just configure NPD to read the kubelet's kubeconfig directly?

Reply:
I figured that it'd be a larger, more confusing/fragile change. This way the used kubeconfig is always in the same location, regardless of NPD configuration